tutor/hooks/catalog.py

```diff
@@ -123,6 +123,12 @@ def run_this_on_start(root, config, name):
     #: :parameter str root: absolute path to the project root.
     PROJECT_ROOT_READY: Action[str] = Action()
 
+    #: Triggered after all interactive questions have been asked.
+    #: You should use this action if you want to add new questions.
+    #:
+    #: :parameter dict config: project configuration.
+    INTERACTIVE_CONFIGURATION: Action[[Config]] = Action()
```

Comment:
Can you please move this action such that it's listed in alphabetical order? Also, we should rename it to `CONFIG_INTERACTIVE`.

Reply:
@regisb Done